.ebextensions/nginx/conf.d/move.mil.conf

```diff
@@ -0,0 +1,46 @@
+upstream move.mil-rails {
+  server unix:///var/run/puma/my_app.sock;
+}
+
+log_format  healthd '$msec"$uri"'
+                    '$status"$request_time"$upstream_response_time"'
+                    '$http_x_forwarded_for';
+
+server {
+  listen 80;
+
+  server_name _ localhost;
+
+  if ($time_iso8601 ~ "^(\d{4})-(\d{2})-(\d{2})T(\d{2})") {
+    set $year $1;
+    set $month $2;
+    set $day $3;
+    set $hour $4;
+  }
+
+  access_log /var/log/nginx/access.log main;
+  access_log /var/log/nginx/healthd/application.log.$year-$month-$day-$hour healthd;
+
+  include extras/directive-only/extra-security.conf;
+  include extras/directive-only/x-ua-compatible.conf;
+  include extras/location/expires.conf;
+  include extras/location/protect-system-files.conf;
```

Comment:
I _think_ these relative paths should work out. They're relative to `/etc/nginx/nginx.conf` which imports this file ( which _should_ be at `/etc/nginx/conf.d/elasticbeanstalk/move.mil.conf` if that Amazon blog post is accurate). We'll find out when we deploy…